modules/engine/src/shader-inputs.ts

```diff
@@ -77,10 +52,10 @@ export class ShaderInputs<
     log.log(1, 'Creating ShaderInputs with modules', Object.keys(modules))();
 
     // Store the module definitions and create storage for uniform values and binding values, per module
-    // @ts-expect-error Fix typings
-    this.modules = modules;
+    // @ts-expect-error
```

Comment:
I still feel that every ts-expect-error should have a comment. i think there is a linter rule that enforces that...